bin/deregister_single_target

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+
+require "aws-sdk-elasticloadbalancingv2"
+
+host_id = ARGV[0]
+
+if host_id.nil?
+  fail "Please specify HOST_ID"
+end
+
+REGION = "us-west-2".freeze
+class AWSClient
+  class << self
+    def elbv2
+      @elbv2 ||= Aws::ElasticLoadBalancingV2::Client.new(region: REGION)
+    end
+  end
+end
+
+tgs = AWSClient.elbv2.describe_target_groups.target_groups
+
+for tg in tgs do
+  targets = AWSClient.elbv2.describe_target_health(target_group_arn: tg.target_group_arn)
+                  .target_health_descriptions.map(&:target)
+  target_ports = targets.map(&:port).uniq
+
+  AWSClient.elbv2.deregister_targets({
+    target_group_arn: tg.target_group_arn,
+    targets: [
+      {
+        id: host_id,
+        port: target_ports[0]
+      }
+    ]
+  })
+end
+
+for tg in tgs do
```

Comment:
Don't use for in ruby, use `each` instead. See https://stackoverflow.com/questions/3294509/for-vs-each-in-ruby